python/ray/_private/storage.py

```diff
@@ -25,7 +26,8 @@
 _filesystem = None
 
 
-@client_mode_hook(auto_init=True)
+@wrap_auto_init
```

Comment:
This api (along with `get_client` and `list_named_actors`) isn't part of `__init__.py:__all__`. 
Not sure if they should be there. I didn't move them to there, but just added a standalone wrapper there.

Reply:
Sounds good. They are not public APIs yet, but the plan is to eventually get them there.